extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -31,6 +32,11 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
   private val onGeoJsonParsedListenerList = mutableListOf<OnGeoJsonParsed>()
   private var ignoreParsedGeoJson = false
 
+  private val workerHandler by lazy {
+    Handler(workerThread.looper)
+  }
+  private val mainHandler = Handler(Looper.getMainLooper())
```

Comment:
btw I guess it could be static, no need to create each time, wdyt?

Reply:
Yes, this one could be static.